RetsTalk/RetsTalk/Retrospect/Model/Retrospect.swift

```diff
@@ -166,6 +167,11 @@ extension Retrospect {
                     format: "userID = %@ AND status = %@ AND isPinned = %@ AND createdAt < %@",
                     argumentArray: [userID, Texts.retrospectFinished, false, lastRetrospectCreatedDate]
                 )
+            case .monthly(let startOfMonth, let startOfNextMonth):
+                CustomPredicate(
+                    format: "userID == %@ AND createdAt >= %@ AND createdAt < %@",
+                    argumentArray: [userID, startOfMonth, startOfNextMonth]
+                )
```

Comment:
이번 달만 필요하다면 다음과 같이 쿼리문을 만들어서 월이 일치하는 것을 만들 수 있을 것 같네요! (동작 확인 해봄)
```suggestion
            case .monthly(let startOfMonth):
                CustomPredicate(
                    format: "userID == %@ AND EXTRACT(MONTH FROM createdAt) == %@",
                    argumentArray: [userID, startOfMonth]
                )
            }
```

~~근데 이 경우 연도가 다른 같은 월의 회고를 가져올 것 같아서 년도도 같이 들어가면 좋을 것 같아요!~~
라고 생각했는데 Date를 불러올 때 년 월까지 불러와서 인자로 넣어주네요! 명칭이 헷갈려서 바꿔봐도 좋을 것 같아요!

Reply:
저런 식으로 바꾸려면 년도 추가 되어야하긴 하겠네요!

Reply:
크리티컬한 이슈네요 반영하겠습니다.

Reply:
저런 포맷이 있었군요.. 일단 후순위 해보겠습니다..ㅎㅎㅎ감사합니다

Reply:
이 부분은 저가 한번 알아보고 이번에 Manager 구현 PR에 추다하도록 하겠습니다!